### ISSUE TITLE
what is "treshold" work for?

### ISSUE THREAD

**Issue by kjxcc** [storyicon/comfyui_segment_anything#20]

**Comment by dnl13** [comment]
filt_mask = logits_filt.max(dim=1)[0] > threshold
The provided code filters out the boxes detected by GroundingDino (by prompt) before passing them to SAM for detailed masking. Specifically, it creates a binary mask (filt_mask) based on the condition that the maximum value along the second dimension of the logits_filt tensor is greater than the specified threshold. Boxes meeting this condition are retained in the filtering process.
